index.html

```diff
@@ -1,64 +1,131 @@
-                <button type="button" class="btn btn-primary">Substract</button>
-            </div>
-            <div>
-                <input name="result" value="0" class="form-control" disabled />
-            </div>
-        </form>
 
+    <!-- Visible header at top of page using Bootstrap -->
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+        <a class="navbar-brand" href="#">Hollis's JS Calculator</a>
+        </div>
+    </nav>
+    <br></br>
 
-    </div>
-    <div>Footer</div>
+    <!-- Allow two numbers to be entered
+show buttons for add and subtract
+        then show the results  -->
+    <!-- Create a form to reference in JS -->
+    <form name="inputs">
+        <!-- Create our card to hold our calculator controls and center it -->
+        <div class="card bg-light mb-3" style="max-width: 90%;margin:0 auto;">
+            <h4 class="card-header">Main Calculator</h4>
+            <!-- Input box one-->
+            <div class="input-group mb-3" style="max-width: 95%;margin:0 auto;padding-top: 15px;">
+                <div class="input-group-prepend">
+                    <span class="input-group-text" id="basic-addon1">Number One:</span>
+                </div>
+                <input type="number" pattern="[0-9]*" class="form-control" placeholder="" aria-label="Number One"
+                    aria-describedby="basic-addon1" name="numberOne"></input>
+            </div>
 
+            <!-- Input box two-->
+            <div class="input-group mb-3" style="max-width: 95%;margin:0 auto;">
+                <div class="input-group-prepend">
+                    <span class="input-group-text" id="basic-addon1">Number Two:</span>
+                </div>
+                <input type="number" pattern="[0-9]*" class="form-control" placeholder="" aria-label="Number Two"
+                    aria-describedby="basic-addon1" name="numberTwo"></input>
+            </div>
 
+            
+                <!-- Create the function buttons -->
+                <div class="btn-group" role="Group" style="padding-bottom: 15px;margin:0 auto;">
+                <button type="button" onclick="calculator.addNumbers()" class="btn btn-primary">Add (+)</button>
+                <button type="button" onclick="calculator.subtractNumbers()" class="btn btn-primary">Subtract
+                    (-)</button>
+                <button type="button" onclick="calculator.multiplyNumbers()" class="btn btn-primary">Multiply
+                    (x)</button>
+                <button type="button" onclick="calculator.divideNumbers()" class="btn btn-primary">Divide (÷)</button>
+                <button type="button" onclick="window.location.reload()" class="btn btn-danger">Clear
+                    Calculator</button>
+            </div>
+            <!-- Display the result! -->
+        <div class="card-footer">
+            <div class="input-group mb-3" style="max-width: 98%;margin:0 auto;">
```

Comment:
Looks great. Not that the style attribute could be removed. The value there could be moved to the style tag with a new css selector or an update to mb-3 or input-group within the style tag in the header.